ledger/complete/ledger.go

```diff
@@ -223,7 +223,7 @@ func (l *Ledger) Set(update *ledger.Update) (newState ledger.State, trieUpdate *
 	l.metrics.UpdateDuration(elapsed)
 
 	if len(trieUpdate.Paths) > 0 {
-		durationPerValue := time.Duration(elapsed.Nanoseconds()/int64(len(trieUpdate.Paths))) * time.Nanosecond
```

Comment:
can you explain why?

Reply:
time.Duration already gives the number in nanoseconds, so multiplying by time.Nanosecond (1) does not matter.